src/bundler.ts

```diff
@@ -79,7 +79,7 @@ class Bundler {
                     String(opts.abspath).trim() !== '') ?
         path.resolve(opts.abspath) :
         undefined;
-    this.pathResolver = new PathResolver(Boolean(this.abspath));
+    this.pathResolver = new PathResolver(this.abspath);
```

Comment:
tl;dr - I think there are two separate but related features, both optional and I propose the names `basePath` and `absPathPrefix` for them.  The filesystem`root` is not provided or used in this context.

---

When we discussed `root` before, it was in the context of the filesystem root (at least that was my understanding) when telling bundler to go bundle.

The thing is that the filesystem's folder root should make no difference to this PathResolver (maybe PathResolver should be renamed to avoid confusion with Analyzer's resolvers), precisely because they have jurisdiction over and may totally redefine relationship of physical files to everything else, where PathResolver honors the resolved url topology.

Besides which, there are no opportunities to apply the filesystem `root` folder concept in PathResolver logic.

However, there might be reason to support two arguments:
 - a path to resolve relative urls to.  typically '/' but could be '/myapp/' etc.
 - a prefix to apply to absolute urls.

The value of separating those options would be to allow expression of absolute hrefs for assets known to be peer-provided in a deployment that would be unaffected by the specific mounting root of the application `/path/to/myapp`.

We could call them `root` and `prefix` but I don't want `root` to be confused with `root` as we use for targeting files to bundle...  Also, applying a prefix to existing absolute paths is a new/separate/different bundler feature from resolving relative paths to an absolute form AFAIK.

Perhaps `basePath` would be a more appropriate term than `root`?  In the sense that `basePath`  applies only to relative paths only whereas having no effect on absolute paths.  Then `absPathPrefix` could be the term used to express the prefix applied to ALL absolute paths.

In this way they could also function independently.